stories/Welcome.mdx

```diff
@@ -1,7 +1,138 @@
+            <p>DevRel Engineer</p>
+          </div>
+          <div>
+            <a href="https://github.com/choir27" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/richard-choir/" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Ryan Furrer</h4>
+            <p>UX Engineer</p>
+          </div>
+          <div>
+            <a href="https://github.com/ryandotfurrer" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/ryanfurrer/" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Walter Furrer</h4>
+            <p>Documentation Engineer</p>
+          </div>
+          <div>
+            <a href="https://github.com/FurrerW" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/furrerw/" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Shashi Lo</h4>
+            <p>Engineering Manager</p>
+          </div>
+          <div>
+            <a href="https://github.com/shashilo" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/shashilo/" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Corina Murg</h4>
+            <p>Frontend Engineer</p>
+          </div>
+          <div>
+            <a href="https://github.com/CorinaMurg" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/corinamurg/" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Chris Nowicki</h4>
+            <p>Software Engineer</p>
+          </div>
+          <div>
+            <a href="https://github.com/chris-nowicki" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/chris-nowicki/" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Mai Vang</h4>
+            <p>Frontend Engineer</p>
+          </div>
+          <div>
+            <a href="https://github.com/vmaineng" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/mai-vang-swe/" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>
+      </section>
+    </section>
 
-<div className="doc_nav_links">
-  <p>[Application Setup](?path=/docs/about-application-setup--docs)</p>
-</div>
```

Comment:
Are we not linking to other pages of the documentation?

Reply:
To be honest, I completely forgot about this. I will add links to other parts of the documentation tonight.

Reply:
Added button to the Application Setup page. Working with Ryan tomorrow on final styling.